DXRMapFixups/DeusEx/Classes/DXRFixupParis.uc

```diff
@@ -83,6 +84,31 @@ function PreFirstEntryMapFixes()
             }
 
             SetAllLampsState(false, false, false); // surely Nicolette didn't leave all the lights on when she moved out
+
+            foreach AllActors(class'FlagTrigger', ft) {
+                if (ft.FlagName == 'ChateauInCellar') {
+                    ft.Destroy();
+                    break;
+                }
+            }
+
+            ft = Spawn(class'FlagTrigger',,, vect(1364.082031, 2048.021240, -311.900421));
```

Comment:
either use `spawnm` or `vectm`, `vectm` is slightly more convenient because the spawn function gets special treatment from UCC to avoid manual casting

```suggestion
            ft = Spawn(class'FlagTrigger',,, vectm(1364.082031, 2048.021240, -311.900421));
```